### PR TITLE
Add supserset manifests

### PR DESCRIPTION
**Pull request by crobby**
Adding the super set manifests to odh. For testing please follow directions here: https://gitlab.com/opendatahub/opendatahub-operator/-/blob/master/docs/deploying-superset.adoc
Skip the ODH installation parts

### CODE REVIEW

**Comment by crobby** [comment]
This one isn't quite working yet.  Having trouble getting my vars to get replaced.

**Comment by crobby** [comment]
> @crobby is this still WIP, it has AI library everywhere, copy-->paste eh ???

No, this branch was built on top of my other PR for ai-library...once that merges, this will sit on top of it cleanly and only show the superset diffs.

**Comment by nakfour** [comment]
@crobby  for superset we had a lot of configurable parameters
that are not populated. https://gitlab.com/opendatahub/opendatahub-operator/-/blob/master/deploy/crds/opendatahub_v1alpha1_opendatahub_cr.yaml

**Comment by nakfour** [comment]
as @crobby mentioned the default user/password to the portal is admin/admin

**Comment by crobby** [comment]
@nakfour I made a tweak to how the admin password works.  Can now be specified as a param in kfctl_openshift.yaml.  The net result is that we don't need the secret.yaml at all and we can just generate the necessary secret using kustomize's secretGenerator functionality.

File: superset/base/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (vpavlin) [inline superset/base/kustomization.yaml]: Is there any specific reason why the labels are specified in each resource rather than in kustomization.yaml?

Comment (crobby) [reply superset/base/kustomization.yaml]: Turns out that a side-effect of "commonLabels" is that the services also wind-up with the labels and that has a negative effect when mapping a route to a service/pod.  https://github.com/kubernetes-sigs/kustomize/issues/1009  I was not able to get the metadataLabelTransformer workaround to work.

Comment (crobby) [reply superset/base/kustomization.yaml]: I have created a tracking issue to remind us (me) to go back and use the labelTransformer when we have a released kfctl that supports it.  https://github.com/opendatahub-io/odh-manifests/issues/7

**Pull request merged**